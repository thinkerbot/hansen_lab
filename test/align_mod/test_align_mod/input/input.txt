AAAQAAA	0.0004000.0	1	with	extra	fields
QBBBBBB	0.4000000.0	2
CCCCCCQ	0.0000004.0
DDDDDDD	0.0000000.0	4
EEEEEEE	0.0001000.0	5
FFQFQFF	0.0040400.0	6
GGQGQGG	0.0041400.0	7