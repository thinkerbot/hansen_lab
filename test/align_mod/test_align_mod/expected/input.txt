|AAA|Q|AAA|1|with|extra|fields|
|.|Q|BBBBBB|2|
|CCCCCC|Q|.|
|FF|Q|FQFF|6 (1)|
|FFQF|Q|FF|6 (2)|
|GG|Q|GQGG|7 (1)|
|GGQG|Q|GG|7 (2)|
